devtools/common/src/main/java/io/quarkus/cli/commands/CommandsException.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.cli.commands;
+
+public class CommandsException extends Exception {
+
+    /**
+     * 
+     */
+    private static final long serialVersionUID = 1L;
+
+    public CommandsException(String message) {
```

Comment:
Hmmm? I'm not sure about this, you already have a `CommandException` that is properly thrown from https://github.com/quarkusio/quarkus/pull/2252/files#diff-8c59d13ed86a532b9ea037d224b29343R35 so you wouldn't have to add it in the catch here: https://github.com/quarkusio/quarkus/pull/2252/files#diff-8c59d13ed86a532b9ea037d224b29343R47 .

But maybe I'm missing something?

Reply:
CommandException is from org.aesh.command but the devtools common don't have dep on it. not sure it's a good idea
So yes I can use it and add the dep if you want.

Reply:
Ah OK, I missed that. Let's use a `IllegalArgumentException` then. I think it matches quite well what we do.